### PR TITLE
Fix metadata for charge.php

### PR DESCRIPTION
**Pull request by ptibogxiv**
Fix for dol_id, dol_type

### CODE REVIEW

File: htdocs/public/payment/newpayment.php [hunk 1]
```diff
@@ -1531,6 +1544,7 @@ class="stripe-button"
 		print '<input type="hidden" name="forcesandbox" value="'.GETPOST('forcesandbox','alpha').'" />';
 		print '<input type="hidden" name="email" value="'.GETPOST('email','alpha').'" />';
 		print '<input type="hidden" name="thirdparty_id" value="'.GETPOST('thirdparty_id','int').'" />';
+    print '<input type="hidden" name="thirdparty_id" value="'.GETPOST('thirdparty_id','int').'" />';
```
Comment (eldy) [inline htdocs/public/payment/newpayment.php]: This line is doubled ?

File: htdocs/public/payment/newpayment.php [hunk 2]
```diff
@@ -1543,7 +1557,7 @@ class="stripe-button"
 	    <div id="card-element">
 	    <!-- a Stripe Element will be inserted here. -->
 	    </div>
-
+      <input name="savesource" type="checkbox"> '.$langs->trans("SaveSource").'
```
Comment (eldy) [inline htdocs/public/payment/newpayment.php]: What is this box for ? What happen if user check this box ?

File: htdocs/public/payment/newpayment.php (hunk 1 above)
Comment (ptibogxiv) [reply htdocs/public/payment/newpayment.php]: bad paste 👎

File: htdocs/public/payment/newpayment.php (hunk 2 above)
Comment (ptibogxiv) [reply htdocs/public/payment/newpayment.php]: with rgpd, we can only save for later source/card with an explicit customer's agreement
i will add the code to use it

**Pull request merged**